src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment.component.html

```diff
@@ -29,39 +33,29 @@
-                    <div *ngIf="!referencedFeedback || referencedFeedback.length == 0" class="col-12 col-lg-8 col-xl-6">
-                        <div class="alert alert-secondary text-center" role="alert">
-                            <p>{{ 'artemisApp.fileUploadAssessment.assessInstruction' | translate }}</p>
-                        </div>
-                    </div>
-                    <div *ngFor="let assessment of referencedFeedback; let i = index" class="col-12 col-lg-6 col-xl-6">
-                        <jhi-assessment-detail
-                            [(assessment)]="referencedFeedback[i]"
-                            (assessmentChange)="updateAssessment()"
-                            (deleteAssessment)="deleteAssessment($event)"
-                            [readOnly]="readOnly"
-                        ></jhi-assessment-detail>
-                    </div>
-                    <div class="col-12">
-                        <button class="btn btn-success" (click)="addFeedback()" [disabled]="readOnly">
-                            {{ 'artemisApp.fileUploadAssessment.addFeedback' | translate }}
-                        </button>
-                    </div>
-                </div>
-                <jhi-assessment-general-feedback [(feedback)]="generalFeedback" (feedbackChange)="validateAssessment()" [readOnly]="readOnly"></jhi-assessment-general-feedback>
-            </ng-container>
         </div>
-        <!--endregion-->
-
         <!--region Right Panel-->
         <fa-icon right-header [icon]="['far', 'list-alt']"></fa-icon>&nbsp;
         <span right-header jhiTranslate="assessmentInstructions.instructions.instructions">Instructions</span>
         <div right-body class="alert alert-info text-center mb-4" role="alert" jhiTranslate="artemisApp.textAssessment.assessmentInstruction"></div>
         <jhi-assessment-instructions right-body [exercise]="exercise" class="markdown-preview"></jhi-assessment-instructions>
         <!--endregion-->
     </jhi-resizeable-container>
-</jhi-assessment-layout>
+    <!--Feedbacks-->
+    <div class="row mt-3">
+        <div class="col-md-6">
+            <div *ngIf="invalidError" class="col-8 alert alert-danger" role="alert">{{ invalidError | translate }}</div>
+            <jhi-unreferenced-feedback [(feedbacks)]="unreferencedFeedback" (feedbacksChange)="validateAssessment()" [readOnly]="readOnly"></jhi-unreferenced-feedback>
+            <jhi-assessment-general-feedback [(feedback)]="generalFeedback" (feedbackChange)="validateAssessment()" [readOnly]="readOnly"></jhi-assessment-general-feedback>
+        </div>
+    </div>
+</ng-template>
+
+<ng-template #noSubmission>
+    <div class="alert alert-warning text-center mt-4" role="alert">
+        <p jhiTranslate="artemisApp.textAssessment.notFound">We haven't found any new text submission without an assessment, please go back.</p>
```

Comment:
I think this should also be file upload

Reply:
you are right, I also missed this one
```suggestion
        <p jhiTranslate="artemisApp.fileUploadAssessment.notFound">We haven't found any new text submission without an assessment, please go back.</p>
```